src/context/authContext.js

```diff
@@ -0,0 +1,51 @@
+import { createContext, useContext, useState } from 'react';
+import axios from 'axios';
+
+const AuthContext = createContext();
+
+export const useAuth = () => useContext(AuthContext);
+
+export const AuthProvider = ({ children }) => {
+
+    const [user, setUser] = useState(null);
+    const [encodedToken, setEncodedToken] = useState();
+
+    const SignupUser = async (email, password) => {
+        try {
+            const res = await axios.post(`/api/auth/signup`, {
+                firstName: "Shivani",
+                lastName: "Gangadharan",
```

Comment:
Why r u assigning value while sending data it will come from input box  correct me if I misunderstood anything

Reply:
You're right, thanks for pointing it out. Fixed it [5d44d50](https://github.com/shivanigangadharan/streambucket/pull/1/commits/5d44d50992faa27cf26e6bbf0724535aba5b7a15).